crates/primitives-traits/src/constants/mod.rs

```diff
@@ -54,7 +57,35 @@ pub const ETHEREUM_BLOCK_GAS_LIMIT: LazyCell<u64> = LazyCell::new(|| {
 ///
 /// Note that min base fee under different 1559 parameterizations may differ, but there's no
 /// significant harm in leaving this setting as is.
-pub const MIN_PROTOCOL_BASE_FEE: u64 = 7;
+// pub const MIN_PROTOCOL_BASE_FEE: u64 = 7;
+
+pub static MIN_PROTOCOL_BASE_FEE: LazyLock<Arc<RwLock<u64>>> = LazyLock::new(|| {
+    Arc::new(RwLock::new(7))
```

Comment:
could we use atomic u64 here? usually gives us free locks on hardware level

Reply:
I tried using atomics, but I didn't find a way to do the read into a `&u64` without taking the value `into_inner`. Maybe I used it wrong. ?